GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -865,7 +873,36 @@ protected override void OnMouseWheel(MouseEventArgs e)
             }
             else
             {
-                base.OnMouseWheel(e);
+                // Reset unconsumed wheel delta when the mouse wheel is idle.
+                if (_lastMouseWheel.ElapsedMilliseconds > 1500)
```

Comment:
I suggest defining a const instead of using the magic number inline.

Reply:
Agreed. Where would I define it?

Reply:
At the top of the class should be fine.